### DISCUSSION TITLE
What can I do if idea owner is non-responsive?

### DISCUSSION THREAD

**Discussion by chenasraf** [open-source-ideas/ideas#330] [Q&A] [answered]
I've completed an implementation of one issue, and it's been 2 weeks+ of reminder messages and no reply at all.
 
Is there anything to do about it, or is this only up to the issue owner? I would like to add another completion to HoF.

Thanks in advance :)

**Comment by KaKi87** [comment]
I'll look into your project, don't worry. I just have a lot of things going on lately, that's all.

**Reply by chenasraf** [reply]
I just wasn't sure you were aware at all. Sorry for bothering you ❤️

**Comment by FredrikAugust** [comment, marked as answer]
I can answer as the maintainer as well. Generally I think it's nice if the original idea creator approves of the solution that has been implemented, but in cases like this (had it not been for @KaKi87 coming back), I would say you could have just created a PR for adding it to the HoF or tagged me in the issue thread:)
